docs/modules/documentation/containers/alert/examples/alert-example.component.html

```diff
@@ -1,28 +1,15 @@
-<fd-alert [dismissible]="true" [type]="'warning'" (close)="closeAlert($event)" #warningAlert>
-    A dismissible warning type alert.
+<fd-alert [type]="'warning'">
+    A dismissible warning alert.
```

Comment:
For all these alert types, should we put the actually string value of the type in bold or quotes or something?
It never actually lists the values you can provide to "type", just part of the sentence of the example.

Reply:
Not sure I understand what you mean. You want to put the 'warning' in the content in bold?

Reply:
Or something like that, we never actually say what values you can provide for `type`
It should be obvious from the example, but I never trust things to be obvious, would be better if it was explicit :)